pgmetadata/www/pgmetadata.js

```diff
@@ -8,7 +8,6 @@
 */
 
 var lizPgmetadata = function() {
```

Comment:
You should use a [self invoking function](https://sarfraznawaz.wordpress.com/2012/01/26/javascript-self-invoking-functions/) unless you need `lizPgmetadata` elsewhere.